### DISCUSSION TITLE
OSP suite latest version installation error

### DISCUSSION THREAD

**Discussion by OZaher** [Open-Systems-Pharmacology/Forum#1009] [Questions & Problems] [answered]
Hello, 
I am trying to install the latest version of OSP on my university laptop and it gives me an error message with 0x800f081e unspecified error.
The logfile shows and unexplained error while installing the EXE file.

**Comment by msevestre** [comment]
Could you install the previous version without any problems?

**Reply by OZaher** [reply]
I tried the previous version but still gives me the same error

**Reply by msevestre** [reply]
It looks like an issue with your windows installation then.
What system are you using? 
Do you have administration rights on your laptop? Maybe @Yuri05 has some tips as to what to try?

**Reply by OZaher** [reply]
I run it on windows 10
I have administrative rights on the system.
I tried to check the .netframework but still not able to install the OSP

**Reply by msevestre** [reply]
you can always try to use the portable version (no install) in the meantime of PKSim and MoBi
https://github.com/Open-Systems-Pharmacology/PK-Sim/releases/download/v10.0/pk-sim-portable-setup.10.0.256.zip

and 
https://github.com/Open-Systems-Pharmacology/MoBi/releases/download/v10.0.122/mobi-portable-setup.10.0.122.zip

**Reply by OZaher** [reply]
Thanks

**Comment by Yuri05** [comment]
Can you install the following components manually first?
1. [VC++ 2015-2019 redistributable](https://aka.ms/vs/16/release/vc_redist.x64.exe)
2. [.NET Framework 4.7.2](https://go.microsoft.com/fwlink/?LinkID=863265)

Did it work? If yes: try to install OSP Suite 10 again. If not: which of the manual installations failed and what was the error?

**Reply by OZaher** [reply]
Hello Thanks for your reply
.NET Framework installation failed with the following error in the logfile: 
Exe (C:\61bee3b3f907d81c690b\x64-Windows10.0-KB4073120-x64.cab) failed with 0x800f081e - (null).
Final Result: Installation failed with error code: (0x800F081E) (Elapsed time: 0 00:01:02).

**Comment by Yuri05** [comment, marked as answer]
Well, that means your Windows installation is corrupted (e.g. the last installation of Windows update was not successfull).
Nothing we can do here. Look for the error code 0x800F081E for possible solutions.
E.g. https://social.technet.microsoft.com/Forums/windows/en-US/d36d88ed-5b5c-45ea-bbc4-3ff865027af0/can-not-install-net-framework-472-error-hresult-quot0x800f081equot?forum=win10itproapps
